ci/bootstrap.py

```diff
@@ -66,17 +69,17 @@ def __init__(
 async def docker_run(*args: str):
     script = ' '.join([shq(a) for a in args])
     outerr = await check_shell_output(script)
+    print(outerr)
```

Comment:
I can clean it up, or change it to only print errors, but I found the output to be very useful. Otherwise when anything failed it would just say `FAILURE` with no other information.